src/components/Collections/Table/Table.vue

```diff
@@ -344,64 +341,41 @@ export default {
       // if a particular entry in the table is a component or not
       return value.type == "component";
     },
-    initialiseSortOrders() {
-      // initialise sorting orders for all table columns
-      // set it to 1 - ascending
-      const columnSortOrders = {};
-      this.columns.forEach(function (key) {
-        columnSortOrders[key] = 1;
-      });
-      this.sortOrders = columnSortOrders;
-    },
     capitalize(str) {
       return str.charAt(0).toUpperCase() + str.slice(1);
     },
-    sortBy(key) {
-      // invoked when sorting arrows are clicked on a table column
-      // sortOrder value can be 1(ascending) or -1(descending)
-      // everytime this method is involed, sortOrder for the "key" is toggled
-      this.sortKey = key;
-      this.sortOrders[key] = this.sortOrders[key] * -1;
+    sortBy(columnName) {
+      // toggle the sort order for "number_of_viewers" column
+      // and emit it to the parent (Home.vue)
+      if (columnName == "number_of_viewers") {
+        this.numberOfViewersSortOrder = this.numberOfViewersSortOrder * -1;
+        this.$emit("sort-by-number-of-viewers", this.numberOfViewersSortOrder);
```

Comment:
seems too long to me and don't think that is needed.
```suggestion
        this.$emit("sort-num-viewers", this.numberOfViewersSortOrder);
```